script/c100219001.lua

```diff
@@ -0,0 +1,141 @@
+--覇王白竜オッドアイズ・ウィング・ドラゴン
+--Odd-Eyes Wing Dragon
+--Scripted by Eerie Code
+function c100219001.initial_effect(c)
+	c:EnableReviveLimit()
+	aux.EnablePendulumAttribute(c,false)
+	aux.AddSynchroProcedure2(c,aux.FilterBoolFunction(Card.IsAttribute,ATTRIBUTE_DARK),aux.NonTuner(c100219001.matfilter))
+	--atk
+	local e1=Effect.CreateEffect(c)
+	e1:SetDescription(aux.Stringid(100219001,0))
+	e1:SetCategory(CATEGORY_TODECK)
+	e1:SetType(EFFECT_TYPE_FIELD+EFFECT_TYPE_TRIGGER_O)
+	e1:SetCode(EVENT_BATTLE_CONFIRM)
+	e1:SetRange(LOCATION_PZONE)
+	e1:SetCountLimit(1)
+	e1:SetCondition(c100219001.atkcon)
+	e1:SetOperation(c100219001.atkop)
+	c:RegisterEffect(e1)
+	--disable
+	local e2=Effect.CreateEffect(c)
+	e2:SetDescription(aux.Stringid(100219001,1))
+	e2:SetCategory(CATEGORY_DISABLE)
+	e2:SetType(EFFECT_TYPE_IGNITION)
+	e2:SetProperty(EFFECT_FLAG_CARD_TARGET)
+	e2:SetRange(LOCATION_MZONE)
+	e2:SetCountLimit(1,100219001)
+	e2:SetTarget(c100219001.distg)
+	e2:SetOperation(c100219001.disop)
+	c:RegisterEffect(e2)
+	--Destroy
+	local e3=Effect.CreateEffect(c)
+	e3:SetDescription(aux.Stringid(100219001,2))
+	e3:SetCategory(CATEGORY_DESTROY)
+	e3:SetType(EFFECT_TYPE_QUICK_O)
+	e3:SetCode(EVENT_FREE_CHAIN)
+	e3:SetRange(LOCATION_MZONE)
+	e3:SetCountLimit(1,100219001+100)
+	e3:SetCondition(c100219001.descon)
+	e3:SetTarget(c100219001.destg)
+	e3:SetOperation(c100219001.desop)
+	c:RegisterEffect(e3)
+	--pendulum
+	local e6=Effect.CreateEffect(c)
+	e6:SetDescription(aux.Stringid(100219001,3))
+	e6:SetType(EFFECT_TYPE_SINGLE+EFFECT_TYPE_TRIGGER_O)
+	e6:SetCode(EVENT_DESTROYED)
+	e6:SetProperty(EFFECT_FLAG_DELAY)
+	e6:SetCondition(c100219001.pencon)
+	e6:SetTarget(c100219001.pentg)
+	e6:SetOperation(c100219001.penop)
+	c:RegisterEffect(e6)
+end
+function c100219001.matfilter(c)
+	return c:IsCode(101001039,82044279,100219001,70771599) or c:IsSetCard(0x1ff)
```

Comment:
Odd-Eyes Wing Dragon is not a `Clear Wing` monster
http://yugioh.wikia.com/wiki/Clear_Wing
also it's best to incorporate the setcode into the existing Clear Wing cards now instead of hard coding them in this card.
List of Clear Wing cards
`Clear Wing Fast Dragon, Clear Wing Synchro Dragon, Supreme King Servant Dragon Clear Wing`